src/chrome/background/message-handler.ts

```diff
@@ -43,6 +43,32 @@ export const BackgroundMessageHandler =
     sendMessageWithConfirmation: SendMessageWithConfirmation,
   ): MessageHandlerOutput => {
     switch (message?.discriminator) {
+      case messageDiscriminator.log:
+      case messageDiscriminator.offscreenLog: {
+        const date = message.log._meta.date
+
+        delete message.log._meta
+        const key = `CE_LOG_${date}`
+        try {
+          chrome.storage.session.set({
```

Comment:
we might want to limit the number of log items stored. I think we should restrict it to the last 1000 items